development/docker-compose/readme.md

```diff
@@ -35,53 +35,47 @@ For the setup, you must have the following:
     brew install colima docker docker-compose docker-credential-helper
     colima start --cpu 4 --memory 16
     ```
-   Workaround to fix Colima issue [#764](https://github.com/abiosoft/colima/issues/764) in order to build the Identity Auth Server image using docker compose:
-   ```shell
-    docker buildx create --driver-opt 'image=moby/buildkit:rootless' 
-    ```
    > **NOTE**: Installing Colima is only for macOS. For Windows-based systems, you can install Docker Desktop and run it to start the Docker service before going to the next step.
 2. Log in to the Backbase repo:
     ```shell
     docker login repo.backbase.com
     ```
-
-### Set up the local environment
-
-1. View a list of all the running containers, with their status and configuration:
-    ```shell
-    docker ps
-    ```
-2. To set the Docker image for the version of Edge you are running, replace `2022.09.1` with the value of `BB_VERSION` in the [development/docker-compose/.env](https://github.com/backbase/local-backend-setup/blob/main/development/docker-compose/.env) file.:
+3. Test your setup:
     ```shell
-    docker pull repo.backbase.com/backbase-docker-releases/edge:`2022.09.1`
+    docker pull repo.backbase.com/backbase-docker-releases/edge:2023.04
```

Comment:
<BB_BOM_VERSION> - Can be mentioned 'use latest BB_BOM_VERSION'

Reply:
it's just for testing the access to repo.backbase.com and having proper docker daemon installed, so does not matter which version they are pulling